public/locale/en.json

```
@@ -2033,6 +2043,7 @@
   "state": "State",
   "state_reason_for_archiving": "State reason for archiving <strong>{{name}}</strong> file?",
   "status": "Status",
+  "status_placeholder": "Select diagnosis status",
```

this could be confusing. key names should be specific to the context and usage. `status` alone is ambiguous.

```suggestion
  "diagnosis_status_placeholder": "Select diagnosis status",
```